common/lib/xmodule/xmodule/js/spec/problem/edit_spec.coffee

```diff
@@ -187,21 +187,98 @@ describe 'MarkdownEditingDescriptor', ->
 
 
         </problem>""")
-    it 'markup with multiple answers doesn\'t break numerical response', ->
+    it 'markup with additional answer does not break numerical response', ->
       data =  MarkdownEditingDescriptor.markdownToXml("""
         Enter 1 with a tolerance:
         = 1 +- .02
-        or= 2 +- 5%
+        or= 2
         """)
       expect(data).toXMLEqual("""<problem>
         <numericalresponse answer="1">
           <p>Enter 1 with a tolerance:</p>
-        <responseparam type="tolerance" default=".02"/>
+          <responseparam type="tolerance" default=".02"/>
+          <additional_answer answer="2"/>
           <formulaequationinput/>
         </numericalresponse>
 
+        </problem>"""
+      )
+    it 'markup for numerical with multiple additional answers renders correctly', ->
+      data =  MarkdownEditingDescriptor.markdownToXml("""
+        Enter 1 with a tolerance:
+        = 1 +- .02
+        or= 2
+        or= 3
+        """)
+      expect(data).toXMLEqual("""<problem>
+        <numericalresponse answer="1">
+          <p>Enter 1 with a tolerance:</p>
+          <responseparam type="tolerance" default=".02"/>
+          <additional_answer answer="2"/>
+          <additional_answer answer="3"/>
+          <formulaequationinput/>
+        </numericalresponse>
 
-        </problem>""")
+        </problem>"""
+      )
+    it 'Do not render ranged additional answers for numerical response', ->
+      data =  MarkdownEditingDescriptor.markdownToXml("""
+        Enter 1 with a tolerance:
+        = 1 +- .02
+        or= 2
+        or= [5,7]
+        or= 3
+        """)
+      expect(data).toXMLEqual("""<problem>
+        <numericalresponse answer="1">
+          <p>Enter 1 with a tolerance:</p>
+          <responseparam type="tolerance" default=".02"/>
+          <additional_answer answer="2"/>
+          <additional_answer answer="3"/>
+          <formulaequationinput/>
+        </numericalresponse>
+
+        </problem>"""
+      )
+    it 'Do not add tolerance to additional answers for numerical response', ->
+      data =  MarkdownEditingDescriptor.markdownToXml("""
+        Enter 1 with a tolerance:
+        = 1 +- .02
+        or= 2
+        or= 3 +- .01
```

Comment:
I thought may be we would want the additional answer but not it's tolerance part, that's why we ignored it. But, I think, you are right, it should be consistent with ranges which is entirely ignored. I will update this.